packages/scripts/scripts/env/update.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+const commandExistsSync = require( 'command-exists' ).sync;
+const request = require( 'request' );
+const DecompressZip = require( 'decompress-zip' );
+
+/**
+ * Node dependencies.
+ */
+const { execSync } = require( 'child_process' );
+const { env, cwd, stdout } = require( 'process' );
+const { normalize } = require( 'path' );
+const { createWriteStream } = require( 'fs' );
+const { tmpdir } = require( 'os' );
+
+env.WP_DEVELOP_DIR = cwd() + '/wordpress';
+
+if ( commandExistsSync( 'git' ) ) {
+	execSync( 'git pull', { cwd: normalize( env.WP_DEVELOP_DIR ), stdio: 'inherit' } );
+	buildWordPress();
+} else {
+	stdout.write( "Git isn't available. Switching to downloading a zip version.\n" );
+	const tmpZip = normalize( tmpdir() + '/wordpress-develop.zip' );
+	const tmpZipWriter = createWriteStream( tmpZip );
+
+	// Set up the unzipper to unzip the archive when it finishes downloading.
+	tmpZipWriter.on( 'finish', () => {
+		const unzipper = new DecompressZip( tmpZip );
+
+		unzipper.on( 'extract', buildWordPress );
+
+		stdout.write( 'Extracting...\n' );
+
+		unzipper.extract( {
+			path: normalize( env.WP_DEVELOP_DIR ),
+			strip: 1,
+			filter: ( file ) => file.type !== 'Directory',
+		} );
+	} );
+
+	stdout.write( 'Downloading...\n' );
+	// Download the archive.
+	request( 'https://github.com/WordPress/wordpress-develop/archive/master.zip' ).pipe( tmpZipWriter );
+}
```

Comment:
This is the same as what's in `scripts/env/install.js`. Would it make sense to refactor it out into a `util` function?

Reply:
Refactoring is fun for the whole family! 833f76328